src/libsync/configfile.cpp

```diff
@@ -863,7 +864,16 @@ void ConfigFile::setNewBigFolderSizeLimit(bool isChecked, qint64 mbytes)
 
 bool ConfigFile::confirmExternalStorage() const
 {
-    return getValue(confirmExternalStorageC, QString(), true).toBool();
+    const QVariant fallback = getValue(confirmExternalStorageC, QString(), true);
+    QVariant value = getPolicySetting(QLatin1String(confirmExternalStorageC), fallback);
```

Comment:
Should be const. Also (optional), you could use `const auto` for both `fallback` and `value`.